test/chi/components/header.pug

```diff
@@ -567,45 +514,273 @@ h3 Header with Horizontal Navbar
+      .chi-header__start
+        .-d--flex.-d-lg--none
+          button#drawer-trigger-portal2.chi-button.-icon.-flat.-light.chi-drawer__trigger(data-target='#drawer-1', aria-label='Toggle navigation')
+            .chi-button__content
+              i.chi-icon.-sm--2.icon-menu
+        .chi-dropdown.-d--none.-d-lg--flex
+          button#button-site-menu2.chi-button.-flat.-light.-text--lg.-px--1.-animate.-no-hover.chi-dropdown__trigger App name
+          .chi-dropdown__menu.-inverse.-list.-w--lg
+            a.chi-dropdown__menu-item.-h--auto.-active(href='#')
+              span.chi-dropdown__menu-item_title App name
+              span.chi-dropdown__menu-item_text App description
+            a.chi-dropdown__menu-item.-h--auto(href='#')
+              span.chi-dropdown__menu-item_title App name 2
+              span.chi-dropdown__menu-item_text App description
+            a.chi-dropdown__menu-item.-h--auto(href='#')
+              span.chi-dropdown__menu-item_title App name 3
+              span.chi-dropdown__menu-item_text App description
+      .chi-header__end
+        button.chi-button.-flat.-light.-icon.-no-hover(aria-label='Notifications')
+          .chi-button__content
+            i.chi-icon.icon-bell-outline(aria-hidden='true')
+        button.chi-button.-flat.-light.-icon.-no-hover(aria-label='Support')
+          .chi-button__content
+            i.chi-icon.icon-circle-question-outline(aria-hidden='true')
+        .chi-divider.-vertical.-inverse
+        .chi-dropdown.-d--none.-d-lg--flex
+          button#button-user-menu2.chi-button.-flat.-light.-px--1.-animate.-no-hover.chi-dropdown__trigger(data-position='bottom-end') Menu
+          .chi-dropdown__menu.-inverse.-w--sm
+            a.chi-dropdown__menu-item(href='#')
+              i.chi-icon.icon-user
+              span Item 1
+            a.chi-dropdown__menu-item(href='#')
+              i.chi-icon.icon-logout
+              span Item 2
+        .chi-dropdown.-d--none.-d-lg--flex
+          button#button-eid-menu2.chi-button.-flat.-light.-px--1.-animate.-no-hover.chi-dropdown__trigger(data-position='bottom-end') Menu
+          .chi-dropdown__menu.-inverse.-w--sm
+            a.chi-dropdown__menu-item(href='#')
+              i.chi-icon.icon-user
+              span Item 1
+            a.chi-dropdown__menu-item(href='#')
+              i.chi-icon.icon-logout
+              span Item 2
+  // Drawer
+  
+  .chi-backdrop.-closed
+    .chi-backdrop__wrapper
+      #drawer-1.chi-drawer.-left.-menu.-position--absolute
+        .chi-drawer__header
+          button.chi-button.-icon.-close(aria-label='Close')
+            .chi-button__content
+              i.chi-icon.icon-x
+        .chi-drawer__content
+          .-px--2.-text Drawer content here
+
+
+
+h3 Inverse Portal Header with Impersonation Bar
```

Comment:
@sabah1910 this example is not having correct responsive behaviour yet. You can check the reference here: [https://assets.ctl.io/chi/2.2.0/components/header/#portal-header-with-impersonation-bar](url)